index.html

```diff
@@ -1885,6 +1887,40 @@ <h2>
           shipping option. <a>User agents</a> MAY indicate pending fields in
           the user interface for the payment request.
         </dd>
+        <dt>
+          <dfn>type</dfn> member
+        </dt>
+        <dd>
+          A <a>PaymentItemType</a> enum value. A user agent MAY use the value
+          of <a>type</a> to assist in the presentation of <a>PaymentItem</a>s
+          by, for example, visually grouping types together or otherwise
+          distinguishing them from other types (or from items that have no
+          associated type).
+        </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentItemType">
+      <h2>
+        <dfn>PaymentItemType</dfn> enum
+      </h2>
+      <pre class="idl">
+        enum PaymentItemType {
+          "tax"
+        };
+      </pre>
+      <p>
+        The <a>PaymentItemType</a> serves to categorize a <a>PaymentItem</a>
+        into particular types.
```

Comment:
Should we have some indication of why this feature exists? i.e. Browsers MAY use this to influence their UI

Reply:
+1 - it's always good to say why. :-) I'm not sure if we have user experience data to back it up, but grouping line items into categories (say, national and local taxes) could help users understand what they're being charged.

Reply:
@adrianhopebailie, @stpeter, [above](https://github.com/w3c/payment-request/pull/666/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR1894), I state that the user agent MAY use this to, "for example, visually grouping types together or otherwise distinguishing them from other types (or from items that have no associated type)." 

Is that sufficient?

Reply:
Yes, that seems good enough for now.